Login/Login.css

```diff
@@ -0,0 +1,58 @@
+@import url(https://king-ui.netlify.app/styles.css);
+@import url(https://king-ui.netlify.app/Stylesheets/navbar.css);
+
+@import url(https://king-ui.netlify.app/Stylesheets/reset.css);
+
+body {
+    margin: 0;
+    display: block;
+}
+
+nav {
+    display: flex;
+    justify-content: space-around;
+    border-bottom: 2px solid var(--primary-color);
+    align-items: center;
+    background-color: white;
```

Comment:
would suggest to use variables for colors

Reply:
i think var(--white) would internally trigger white, so why not use white directly?